dev-util/abi_compliance_checker/abi_compliance_checker-2.3.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Binary & source-level backwards compatibility checker"
+DESCRIPTION="ABI Compliance Checker (ACC) is a tool for checking backward \
+binary and source-level compatibility of a C/C++ library. The tool checks \
+header files and shared libraries of old and new versions and analyzes \
+changes in API and ABI (ABI=API+compiler ABI) that may break binary and/or \
+source compatibility."
+HOMEPAGE="http://ispras.linuxbase.org/index.php/ABI_compliance_checker"
+COPYRIGHT="2018 Andrey Ponomarenko's ABI Laboratory"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/lvc/abi-compliance-checker/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="b1e32a484211ec05d7f265ab4d2c1c52dcdb610708cb3f74d8aaeb7fe9685d64"
+SOURCE_DIR="abi-compliance-checker-$portVersion"
+#ADDITIONAL_FILES="example-descriptors"
```

Comment:
Oops, I missed this one :)
```diff
-#ADDITIONAL_FILES="example-descriptors"
 PATCHES="abi_compliance_checker-$portVersion.patchset"
+#ADDITIONAL_FILES="example-descriptors"
```
Links:
* [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Reply:
Missed it too ;)